### DISCUSSION TITLE
Inflight Debouncer

### DISCUSSION THREAD

**Discussion by es0329** [dropbox/Store#459] [Q&A]
I'd like to study the logic that throttles duplicate requests. I love Store but am not allowed to add the dependency into this specific project. My high-level understanding is that it's made possible by using the hashes of requests as keys, but the remainder of that plumbing is still a mystery to me.

I also saw an approach for [request cancelling via OkHttp's `Dispatcher`](https://stackoverflow.com/a/35964678), but those pieces haven't worked in this injection graph or I (most likely) overlooked something.

Thanks for any insight, and keep up the great work. 👍🏽

**Comment by matt-ramotar** [comment]
Hey! Sorry to be slow. Thanks for reaching out. I can definitely shed light on Store5 but will defer to @digitalbuddha @yigit and others on Store4. Will circle back later today

**Comment by matt-ramotar** [comment]
Their talk at KotlinConf might go into it https://www.youtube.com/watch?v=raWdIwsDe-g
